modules/nlu-testing/src/backend/api.ts

```diff
@@ -181,10 +195,27 @@ function results2CSV(tests: Test[], results: _.Dictionary<TestResult>) {
   return stringify(records)
 }
 
-function isRunningFromSources(): string | undefined {
+async function isRunningFromSources(bp: typeof sdk, botId: string): Promise<string | undefined> {
   try {
+    const botConfig = (await bp.bots.getBotById(botId)) as BPDS_BotConfig
+    const bpdsId = botConfig.bpdsId
```

Comment:
what if bpdsId is falsy ? aka someone is running nlu-testing on its own bot. will that crash ? we should return straight up

Reply:
yes!